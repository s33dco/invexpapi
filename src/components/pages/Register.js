import React, { Fragment } from 'react';

const Register = () => {
	return (
		<Fragment>
			<p>Register</p>
		</Fragment>
	);
};

export default Register;
