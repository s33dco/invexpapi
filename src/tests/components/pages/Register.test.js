import React from 'react';
import { shallow } from 'enzyme';
import Register from '../../../components/pages/Register';

let wrapper;

beforeEach(() => {
	wrapper = shallow(<Register />);
});

describe('Register component', () => {
	it('should render Register component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
