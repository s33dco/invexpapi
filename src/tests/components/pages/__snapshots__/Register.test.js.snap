// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register component should render Register component correctly 1`] = `ShallowWrapper {}`;
